### PR TITLE
Make next button grab focus after page completion

### PR DESCRIPTION
**Pull request by 13r0ck**
When the suggested action changes from "Skip" -> "Next" it will now also grab focus.

The best example of this is when configuring WiFi while setting up Pop!_OS. Now after the wifi connection is successful, the user can press `Enter` and continue, rather than just using the mouse or pressing `Tab` a bunch.

### CODE REVIEW

**Review by jacobgkau — APPROVED** [review]
Great improvement. Helps with the time zone selection page, too.

File: gnome-initial-setup/gis-assistant.c [hunk 1]
```diff
@@ -232,6 +232,7 @@ update_navigation_buttons (GisAssistant *assistant)
 
       if (gis_page_get_complete (page)) {
         set_suggested_action_sensitive (next_widget, TRUE);
+	gtk_widget_grab_focus (next_widget);
         set_navigation_button (assistant, next_widget);
```
Comment (jackpot51) [inline gnome-initial-setup/gis-assistant.c:236]: Looks like indentation does not match, make sure to use the same type as the source file

Comment (13r0ck) [reply gnome-initial-setup/gis-assistant.c:236]: Fixed! I will pay more attention to that in the future. Thanks!

**Comment by jacobgkau** [comment]
Improvement's still working on 0d511ba.

It does still look like the indentation's off, though? The new line was too far to the left before, now it's too far to the right. Looks like there's a tab and six spaces where there should be eight spaces.

File: gnome-initial-setup/gis-assistant.c (hunk 1 above)
Comment (13r0ck) [reply gnome-initial-setup/gis-assistant.c:236]: I actually have no idea what is going on here. Using the github IDE, and VS Code I can't get it to line up.

Comment (13r0ck) [reply gnome-initial-setup/gis-assistant.c:236]: Nvm finally got it

**Comment by 13r0ck** [comment]
I finally got it fixed in emacs :facepalm:  . @leviport is right emacs is better.

**Review by jacobgkau — APPROVED** [review]
5c873e5 still looks good.

**Review by jackpot51 — APPROVED** [review]

**Pull request merged**